app/services/mapping/goodsshipment/DestinationBuilder.scala

```diff
@@ -32,6 +33,13 @@ object DestinationBuilder {
       .map(data => createExportCountry(data.countryOfDestination))
       .orNull
 
+  def buildThenAdd(exportsCacheModel: ExportsCacheModel, goodsShipment: GoodsShipment) =
+    exportsCacheModel.locations.destinationCountries.foreach { destinationCountries =>
+      {
```

Comment:
I think we do not need these curly braces here

Reply:
changed